src/ros-vision-interaction/src/interaction-engine/.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: python
```

Comment:
Maybe use this as a submodule, rather than checking the whole repository in - certain things like this file are misleading. Checking all of this into the repository will also lead to a gradual divergence of the two repos.